devtools/common/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -0,0 +1,29 @@
+// this block is necessary to make enforcedPlatform work for Quarkus plugin available
+// only locally (snapshot) that is also importing the Quarkus BOM
+buildscript {
+    repositories {
+        mavenLocal()
+    }
+    dependencies {
+        classpath "io.quarkus:quarkus-gradle-plugin:${quarkus_version}"
+    }
+}
```

Comment:
BTW, we might not need this anymore with newer versions of Gradle. It's not critical though for this specific PR.

Reply:
ok